common/types/offer.go

```diff
@@ -22,6 +22,12 @@ var (
 	// CurOfferVersion is the latest supported version of a serialised Offer struct
 	CurOfferVersion, _ = semver.NewVersion("1.0.0")
 
+	// Don't allow offers over 1000 XMR. Mainly to prevent fat-finger errors, it
+	// could be raised if users need it.
+	maxOfferValue = apd.New(1, 3) // 1000 XMR
```

Comment:
any reason for this number?

Reply:
I was just looking for a semi-round number that's greater than what anyone, at the current time at least, would realistically be swapping. At today's price, 1000 XMR is around $155,410 USD. If you have a different suggestion, let me know and I'll plug it in. I could have just as easily picked 10,000 XMR.